mozilla-services/tecken
# Update js requirements

This updates a few things that GitHub said are security issues as well
as a few things "yarn audit" was sad about, too.

Update dependencies:

* date-fns 2.0.1 -> 2.1.0
* react-scripts 3.0.1 -> 3.1.1

Update resolution things:

* babel-eslint 10.0.1 -> 10.0.2
* braces 2.3.1 -> 3.0.2
* handlebars 4.0.13 -> 4.2.0

Add new resolution things:

* eslint-utils >= 1.4.1 (security issue)
* mixin-deep >= 2.0.1 (security issue)
* set-value >= 3.0.1 (security issue)

To test:

1. `make build`
2. `make setup`
3. `make lintcheck` to make sure there aren't linting issues
4. `make run` and then go to `localhost:3000` and click around
